homeassistant/components/tomorrowio/sensor.py

```diff
@@ -309,6 +312,20 @@ def convert_ppb_to_ugm3(molecular_weight: int | float) -> Callable[[float], floa
         name="Fire Index",
         icon="mdi:fire",
     ),
+    TomorrowioSensorEntityDescription(
+        key=TMRW_ATTR_UV_INDEX,
+        name="UV Index",
+        icon="mdi:sun-wireless",
+    ),
+    TomorrowioSensorEntityDescription(
+        key=TMRW_ATTR_UV_HEALTH_CONCERN,
+        name="UV Radiation Health Concern",
+        value_map=UVDescription,
+        device_class=SensorDeviceClass.ENUM,
+        options=["high", "low", "moderate", "very_high", "extreme"],
```

Comment:
instead of using the strings here, should we reference the `UVDescription` enums instead?

Reply:
> instead of using the strings here, should we reference the `UVDescription` enums instead?

Is that possible? options requires type `list[str]` and UVDescription is an `int` enum. All other options in other modules also use strings.

Reply:
you could do `UVDescription.LOW.name.lower()` but I am not sure if that's better

Reply:
Oh, right. Mmh. I'm not sure, seems to make it less readable.

Reply:
other option is:
`[val.name.lower() for val in UVDescription]`

Really this should be handled in `__post_init__` in `TomorrowioSensorEntityDescription` if `value_map` isn't `None`. You can also set the device class to ENUM using the same logic

Reply:
OK given this is how it's handled, we can merge as is and open a separate PR to make this more generic